src/game-support/hellblade2.md

```diff
@@ -0,0 +1,9 @@
+# Senua’s Saga: Hellblade II
+<!-- script:Aliases [] -->
```

Comment:
Can you add an alias here too?
- `Senua’s Saga: Hellblade 2`
- `Senua’s Saga Hellblade II`
- `Senua’s Saga Hellblade 2`
- `Senua's Saga: Hellblade II`
- `Senua's Saga: Hellblade 2`
- `Senua's Saga Hellblade II`
- `Senua's Saga Hellblade 2`.